packages/e2e-tests/specs/editor/blocks/list.test.js

```diff
@@ -113,6 +113,7 @@ describe( 'List', () => {
 		await clickBlockAppender();
 		await page.keyboard.press( 'Enter' );
 		await page.keyboard.type( '* ' );
+		await page.evaluate( () => new Promise( window.requestIdleCallback ) );
```

Comment:
@youknowriad I found the issue I think. Feel free to include in your PR. I don't mind which PR we go with. We need to wait for this:

https://github.com/WordPress/gutenberg/blob/3da717b8d0ac7d7821fc6d0475695ccf3ae2829f/packages/block-editor/src/store/controls.js#L24-L36